voxblox_msgs/msg/Layer.msg

```diff
@@ -9,5 +9,11 @@ uint8 action   # See action defines below
 voxblox_msgs/Block[] blocks
 
 # Action definitions
-uint8 ACTION_FULL_MAP = 0
+# Set the layer to the state described by this message.
+uint8 ACTION_RESET = 0
```

Comment:
Hmm... Are you use reset should be the default action? I'd prefer merge as 0 and reset as 2, for instance. that way if it's not set, it does something reasonable.

Reply:
The default action that is set during serialization is KUpdate, but you're right, if someone just initializes the ROS msg the default is reset. On the other hand this is the way it was before, FULL MAP = RESET, I thought you might have rosbags somewhere that need to remain compatible?

Reply:
Not sure there's many rosbags where we actually record the map, and I wouldn't worry about invalidating them. If that's the only concern, tehn I'd prefer to swap` 0 <-> 2`

Reply:
Ok, if you're not concerned about this then I'll change it :)